.gitignore

```diff
@@ -78,7 +78,7 @@ target/
 .ipynb_checkpoints
 
 # for now lets ignore all jupyter notebooks
-*.ipynb
+#*.ipynb
```

Comment:
maybe we should just remove this as its not in use?

Reply:
I'll go ahead and uncomment it. It says to ignore the checkpoints in the lines above so I think we should keep it just to ignore all Jupyter related things